oak_restricted_kernel/src/mm/page_tables.rs

```diff
@@ -267,3 +267,35 @@ impl Translator for RootPageTable {
         self.inner.translate_physical_frame(frame)
     }
 }
+
+/// Wrapper struct that holds the current page table is there one.
+pub struct CurrentPageTable {
```

Comment:
nit: perhaps emphasize the fact that this is the current root page table in the name (e.g. CurrentRootPageTable)